test/factories/compute_resources.rb

```diff
@@ -15,6 +15,10 @@
 
     trait :libvirt do
       provider { 'Libvirt' }
+      # url { 'test:///default' } prevents the error:
+      # Call to virConnectOpen failed: Cannot read CA certificate
+      # '/etc/pki/CA/cacert.pem': No such file or directory
```

Comment:
I'm still not entirely sure about this. When you mock, you shouldn't have a "real" connection so I think I consider it a regression in fog-libvirt that we need to fix.

Reply:
Yeah, it makes sense. I have a little experience with the fog-libvirt, so it will take time to investigate the problem.
Next week I'll have a time.

Reply:
I'd prefer not to wait until next week with green nightlies.

Should we pin fog-libvirt to < 0.13 for the time being? Or merge the (working) fix here and clean up later?

Reply:
@stejskalleos How do you want to proceed based on comment around pinning?

Reply:
I think https://github.com/fog/fog-libvirt/pull/163 is a way to solve that regression and https://github.com/theforeman/foreman/pull/10380 is intended to try that out.

Reply:
https://github.com/theforeman/foreman/pull/10380 tells me that works and no change to the factory is needed if we release it as 0.13.1.

Reply:
I will test it today ASAP and see the results

Reply:
I can confirm that on my setup together with https://github.com/fog/fog-libvirt/pull/163 all tests passed.

```
bundle exec rails test \                                             
./test/controllers/api/v2/compute_resources_controller_test.rb \
./test/controllers/hosts_controller_test.rb \
./test/controllers/key_pairs_controller_test.rb \
./test/models/compute_resources/libvirt_test.rb \
./test/models/host_test.rb \
./test/models/orchestration/compute_test.rb \
./test/models/orchestration/dhcp_test.rb \
./test/unit/compute_resource_host_importer_test.rb
```

Let's see if the CI will be green as well to be sure.

Reply:
Looks good!

@ekohl can you merge and release the fog change?